$OpenBSD: patch-phoenix_pro,v 1.8 2015/11/24 15:02:24 abieber Exp $

- Don't build everything twice, just once takes long enough already.
- Remove bundled boost and use the system-wide version instead.

--- phoenix.pro.orig	Tue Mar 17 07:36:55 2015
+++ phoenix.pro	Mon Nov 16 17:29:07 2015
@@ -31,7 +31,8 @@
 # QTPLUGIN  += qjpeg qsqlite
 
 
-CONFIG += debug_and_release
+CONFIG += release
+LIBS += -Wl,-rpath,${LOCALBASE}/lib/qt4 -lQtSerialPort
 win32 {
 # release build using msvc 2010 needs to use Multi-threaded (/MT) for the code generation/runtime library option
 # release build using msvc 2010 needs to add msvcrt.lib;%(IgnoreSpecificDefaultLibraries) to the linker/no default libraries option
